# contain original detection information
object_detection/Detection2D detection2d
# contain the 3d data
sensor_msgs/PointCloud2 cloud